packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -885,6 +886,14 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
     private readonly dataStores: DataStores;
 
+    /**
+     * True, if we should write the GC tree.
+     * False, if data stores should write GC blobs in their summary tree.
+     */
+    public get shouldWriteGCTree(): boolean {
```

Comment:
maybe name should reflect who is writing it? I.e. note the difference between "should wring container runtime" vs. overall "someone should write it"